bundles/tracker/events/components/EventRouter.tsx

```diff
@@ -6,21 +6,26 @@ import DonateInitializer from '../../donation/components/DonateInitializer';
 import Prize from '../../prizes/components/Prize';
 import Prizes from '../../prizes/components/Prizes';
 import { Routes } from '../../router/RouterUtils';
+import NotFound from '../../router/components/NotFound';
 
 const EventRouter = (props: any) => {
   // TODO: type this better when DonateInitializer doesn't need page-load props
   const { eventId } = props;
 
   return (
     <Switch>
-      <Route exact path={Routes.EVENT_PRIZES(eventId)}>
-        <Prizes eventId={eventId} />
-      </Route>
-      <Route exact path={Routes.EVENT_PRIZE(eventId, ':prizeId')}>
-        {({ match }: RouteComponentProps<{ eventId: string; prizeId: string }>) => (
-          <Prize prizeId={match.params.prizeId} />
-        )}
-      </Route>
+      {window.SWEEPSTAKES_URL && (
+        <Route exact path={Routes.EVENT_PRIZES(eventId)}>
+          <Prizes eventId={eventId} />
+        </Route>
+      )}
+      {window.SWEEPSTAKES_URL && (
+        <Route exact path={Routes.EVENT_PRIZE(eventId, ':prizeId')}>
+          {({ match }: RouteComponentProps<{ eventId: string; prizeId: string }>) => (
+            <Prize prizeId={match.params.prizeId} />
+          )}
+        </Route>
+      )}
```

Comment:
```suggestion
      {window.SWEEPSTAKES_URL && (
        <>
          <Route exact path={Routes.EVENT_PRIZES(eventId)}>
            <Prizes eventId={eventId} />
          </Route>
          <Route exact path={Routes.EVENT_PRIZE(eventId, ':prizeId')}>
            {({ match }: RouteComponentProps<{ eventId: string; prizeId: string }>) => (
              <Prize prizeId={match.params.prizeId} />
            )}
          </Route>
        </>
      )}
```

Whenever I see two identical conditionals next to each other, I get scared that it's a mistake. Wrapping with a fragment is p straightforward to simplify.

Reply:
I tried that first, but I got an error about something trying to access `null`, so I think the Switch didn't like that.